src/mqtt/transport/quic/mqtt_quic.c

```diff
@@ -1601,6 +1598,9 @@ mqtt_quictran_ep_connect(void *arg, nni_aio *aio)
 	mqtt_quictran_ep *ep = arg;
 	int              rv;
 
+	// XXX Reset the d_started flag to allow eatablish multistream fast
+	nni_atomic_flag_reset(&ep->ndialer->d_started);
```

Comment:
why do u put it here?

Reply:
It was put to the callback of connection. But this may lead to nni_dialer_start returning an error if nni_dialer_start is too fast. That's why it is put here.